README.md

```diff
@@ -1,3 +1,17 @@
 # AWS Terminator
 
 An AWS Lambda function for cleaning up AWS resources.
+
+## Run the sanity tests
+
+We use `tox` to run the sanity tests:
+
+``` sh
```

Comment:
If there's a prompt, the console lexer should be used for proper highlighting. Otherwise, drop the prompt.
```suggestion
```console
```